src/PassPlanFilePlugin.groovy

```diff
@@ -13,8 +13,8 @@ class PassPlanFilePlugin implements TerraformPlanCommandPlugin, TerraformApplyCo
 
     @Override
     public void apply(TerraformEnvironmentStage stage) {
-        stage.decorate(PLAN,  stashPlan(stage.getEnvironment()))
-        stage.decorate(APPLY, unstashPlan(stage.getEnvironment()))
+        stage.decorate(PLAN_COMMAND,  stashPlan(stage.getEnvironment()))
+        stage.decorate(APPLY_COMMAND, unstashPlan(stage.getEnvironment()))
```

Comment:
Can you clarify why the stash/unstash needed to be narrowed in this way?   How is it related to outputting?

Reply:
It was an attempt to prevent an ordering conflict between the plugins. Anything regarding a plan file is dependent only on the execution of the plan and apply commands. In my case, I wanted to skip the PLAN_COMMAND itself while still executing the PLAN stage (I need the INIT). With the above plugin bound to the entire PLAN phase, it's still going to execute even in the absence of the plan command executing and try to stash a non-existent plan file.

Reply:
Ah, I see.  Thanks for the clarification.

Reply:
Linking to Issue #152, since it touches on the concern about decorating being overly specific.